homeworks/taron_hakobyan/Homework7/Task2.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class Task2 {
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        int number = scan.nextInt();
+        String[] words = digitToWords(number);
+        for (int i = 0; i < words.length; i++) {
+            System.out.print(words[i] + " ");
+        }
+        System.out.println("If you want to continue press 'Yes'");
+        String count = scan.next();
+        if (count.equals("Yes")) {
+            main(args);
+        } else {
+            System.out.println("End!");
+        }
```

Comment:
overall it's not a good idea to call main recursively, plus you can get stackoverflow exceptions.
We do have cycles for that purpose, here you can just use do while